e2e/poms/task1.ts

```diff
@@ -0,0 +1,190 @@
+        await this.page.getByTestId(TASK1SELECTOR.testIds.formDeleteButton).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.deleteArchiveAlertArchiveCheckbox).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.deleteArchiveAlertDeleteButton).click();
+    }
+
+    async buildFor2() {
+        const contentField = this.page.getByTestId(TASK1SELECTOR.testIds.contentTextField);
+
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addSingleChoiceElement).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.singleChoiceOptionsContainer)).toBeVisible();
+        await contentField.click();
+        await contentField.fill(TASK1TEXT.texts.singleChoice);
+
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addMultiChoiceElement).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.multiChoiceOptionsContainer)).toBeVisible();
+        await contentField.click();
+        await contentField.fill(TASK1TEXT.texts.multipleChoice);
+
+        await this.page.getByTestId(TASK1SELECTOR.testIds.singleChoiceOptionsContainer).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addBulkOptionLink).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.bulkAddOptionsTextarea).fill(TASK1TEXT.texts.options);
+        await this.page.getByTestId(TASK1SELECTOR.testIds.bulkAddOptionsDoneButton).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.formSingleChoiceOption)).toHaveCount(10);
+        await this.page.getByTestId(TASK1SELECTOR.testIds.randomizeSwitchLabel).click();
+
+        const originalOptions = await this.page.getByTestId(TASK1SELECTOR.testIds.singleChoiceOption).allTextContents();
+
+        await this.page.getByTestId(TASK1SELECTOR.testIds.multiChoiceOptionsContainer).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addOptionLink).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addOptionLink).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.formMultipleChoiceOption)).toHaveCount(6);
+        await this.page.getByTestId(TASK1SELECTOR.testIds.hideQuestionToggleLabel).click();
+        await this.page.waitForTimeout(TASK1SELECTOR.timeouts.long);
+
+        await this.page.getByTestId(TASK1SELECTOR.testIds.publishButton).click();
+        const page2Promise = this.page.waitForEvent('popup');
+        await this.page.getByTestId(TASK1SELECTOR.testIds.publishPreviewButton).click();
+        const page2 = await page2Promise;
+
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.formMultipleChoiceOption)).toBeHidden();
+
+        const displayedOptions = await page2.getByTestId(TASK1SELECTOR.testIds.singleChoiceOption).allTextContents();
+        expect(JSON.stringify(originalOptions)).not.toBe(JSON.stringify(displayedOptions));
```

Comment:
You don't have to stringify the strings to compare the arrays. You can use the `.toStrictEqual` assertion.
```suggestion
        expect(originalOptions).not.toStrictEqual(displayedOptions);
```